spec/features/withdrawing/edit_withdrawal_spec.rb

```diff
@@ -33,7 +33,7 @@ def then_i_can_see_the_existing_public_explanation
   def when_i_edit_the_public_explanation
     @new_explanation = "Another explanation"
     converted_explanation = GovspeakDocument.new(@new_explanation, @edition).payload_html
-    body = { type: "withdrawal", explanation: converted_explanation, locale: @edition.locale }
+    body = { type: "withdrawal", explanation: converted_explanation, locale: @edition.locale, unpublished_at: @edition.status.details.withdrawn_at }
```

Comment:
It might be time to split this hash over a number of lines to make it a bit more readable.